_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "The First Five Questions"
+date: 2024-01-15
+year: 2024
+---
+
+I've been talking and writing for several years
+about the need for something that would teach programmers
+the useful basics of economics, politics, sociology, and other social sciences.
+A few days ago I framed it as [the Scratch of social sciences][like-scratch];
```

Comment:
OH what an interesting question. Tbh I think because the social sciences are the science of our human lives, the scratch is actually critically reflecting on our lives through a science approach  - which both muddies and clarifies this. I've used Scratch maybe once, but my understanding is that Scratch succeeds in teaching CS by giving you a project that requires it in a friendly, fun, and relevant way. It doesn't teach you The Language of All Languages or The Rules of All Languages, but it teaches you the processes and outlook needed. So if I think about that and apply it to how IRL professors teach social sciences, I think about how we teach people how to observe and ask questions about life experiences - adding the "how" and "why" questions to observations, then teaching them to answer those questions by linking them back to social science theories and research. Teaching them how to science, so to speak.

Reply:
I'm in agreement that a foundation in social sciences is something most programmers would benefit from, but it's not obvious to me that computer scientists in industry or academia would agree. So the opening question "What could we teach in a one-day workshop?" in some sense also needs to address the question "With whom can we get buy-in to get their team of devs to do this workshop?" I can't see anyone in industry doing this. Perhaps academics might, but they may want to see how this gets them closer to publishing something.

Reply:
My working title is "Why Your World Seems So Messed Up", which I hope will be an effective hook :-)